app/auth/access-token/trusted-issuers/trusted-issuers.js

```diff
@@ -0,0 +1,5 @@
+const config = require('../../../config').authConfig
+
+module.exports = [
+  `https://${config.defraId.tenantName}.b2clogin.com/64c9d4f5-a560-4b65-9004-6d1e5ccee51d/v2.0/`
```

Comment:
Does the last part of this address need to be configurable?

Reply:
yeah definitely needs to be config and it should be available under existing env var `process.env.DEFRA_ID_TENANT`